components/fieldgroups/agentic/conversation-event.schema.json

```diff
@@ -161,13 +33,150 @@
+                }
+              }
+            },
+            "xdm:response": {
+              "title": "Response",
+              "type": "object",
+              "description": "Object representing a conversation response",
+              "properties": {
+                "xdm:source": {
+                  "title": "Source",
+                  "type": "string",
+                  "description": "Source of this element",
+                  "examples": ["end-user", "bot", "canned"]
+                },
+                "xdm:raw": {
+                  "type": "array",
+                  "title": "Raw Text and Purpose",
+                  "description": "Raw text with purpose of the text",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "xdm:text": {
+                        "title": "Text",
+                        "description": "Text",
+                        "type": "string"
+                      },
+                      "xdm:purpose": {
+                        "title": "Purpose",
+                        "description": "Purpose",
+                        "type": "string"
+                      }
+                    }
+                  }
+                }
+              }
+            },
+            "xdm:feedback": {
+              "title": "Feedback",
+              "type": "object",
+              "description": "Object representing a conversation feedback",
+              "properties": {
+                "xdm:source": {
+                  "title": "Source",
+                  "type": "string",
+                  "description": "Source of this element",
+                  "examples": ["end-user", "bot", "canned"]
+                },
+                "xdm:raw": {
+                  "type": "array",
+                  "title": "Raw Text and Purpose",
+                  "description": "Raw text with purpose of the text",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "xdm:text": {
+                        "title": "Text",
+                        "description": "Text",
+                        "type": "string"
+                      },
+                      "xdm:purpose": {
+                        "title": "Purpose",
+                        "description": "Purpose",
+                        "type": "string"
+                      }
+                    }
+                  }
+                },
+                "xdm:rating": {
+                  "title": "Rating",
+                  "type": "object",
+                  "description": "Rating data for the feedback",
+                  "properties": {
+                    "xdm:score": {
+                      "title": "Rating Score",
+                      "type": "number",
+                      "description": "Numerical value of the rating provided by the end-user. For example: 0, 1 might be used for thumbs up/down."
```

Comment:
Numerical value of the rating provided by the end-user ranging from -1.0 (negative) to 1.0 (positive).